tests/network/samba/samba_adcli.pm

```diff
@@ -0,0 +1,64 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: samba test conecting with Active Directory using adcli
+# package: samba adcli samba-winbind krb5-client
+#
+# Maintainer: Marcelo Martins <mmartins@suse.com>
+
+use strict;
+use warnings;
+use base "consoletest";
+use registration 'add_suseconnect_product';
+use repo_tools qw(add_qa_head_repo add_qa_web_repo);
+use testapi;
+use utils;
+use lockapi qw(mutex_create);
+
+sub samba_sssd_install {
+    zypper_call('in  samba adcli samba-winbind krb5-client');
+    #Copy config files enviroment.
+    assert_script_run 'curl -f -v ' . autoinst_url . "/data/supportserver/samba/smb.conf  >/etc/samba/smb.conf";
+    assert_script_run 'curl -f -v ' . autoinst_url . "/data/supportserver/samba/krb5.conf  >/etc/krb5.conf";
+    assert_script_run 'curl -f -v ' . autoinst_url . "/data/supportserver/samba/nsswitch.conf  >/etc/nsswitch.conf";
+    assert_script_run 'curl -f -v ' . autoinst_url . "/data/supportserver/samba/resolv.conf  >/etc/resolv.conf";
+    assert_script_run 'curl -f -v ' . autoinst_url . "/data/supportserver/samba/hosts  >/etc/hosts";
```

Comment:
I would modify those files instead redownloading them.

For example:
```perl
assert_script_run("echo '10.0.2.102 susetest.geeko.com susetest' >> /etc/hosts");
assert_script_run("echo '10.0.2.101 win-r70413psjm4.geeko.com win-r70413psjmn4' >> /etc/hosts");
assert_script_run("echo 'nameserver 10.0.2.101' > /etc/resolv.conf");
assert_script_run("echo 'search geeko.com' >> /etc/resolv.conf");
```

Reply:
ok, it is easy to change